### PR TITLE
Run VIB Helm chart validations on push to main

### PR DESCRIPTION
**Pull request by josvazg**
**Description of the change**

Run VIB validations only when merging or pushing into `main`.

### CODE REVIEW

File: .github/workflows/helm-vib.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Verify Helm Chart
 on:
   workflow_dispatch:
-  pull_request_target:
+  push:
```
Comment (alvneiayu) [inline .github/workflows/helm-vib.yaml:4]: I am a little bit confuse, why we are changing from pull_request_target to push? I mean, we are going to launch this VIB action every time that we are pushing, no? Am I wrong?

Comment (josvazg) [reply .github/workflows/helm-vib.yaml:4]: No, it only triggers when you **push** to `main`, and that ONLY happens on merges, as `main` is protected.

File: .github/workflows/helm-vib-lint.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Lint Helm Chart
```
[the remark this answers is not in the document]
Comment (josvazg) [reply .github/workflows/helm-vib-lint.yaml:1]: You have a point there, maybe using `pull_request_target` is enough as we have tested this pre-merge.

File: .github/workflows/helm-vib.yaml (hunk 1 above)
Comment (alemorcuq) [reply .github/workflows/helm-vib.yaml:4]: I think this is a leftover:

```
bitnami-labs:main
```

Just:

```
branches:
  - main
```

File: .github/workflows/helm-vib-lint.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (josvazg) [reply .github/workflows/helm-vib-lint.yaml:1]: changes to lint only on PRs

File: .github/workflows/helm-vib.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
     branches:
       - main
       - bitnami-labs:main
```
Comment (josvazg) [inline .github/workflows/helm-vib.yaml:7]:
```suggestion
```

File: .github/workflows/helm-vib.yaml (hunk 1 above)
Comment (josvazg) [reply .github/workflows/helm-vib.yaml:4]: fixed

File: .github/workflows/helm-vib-lint.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+on:
+  workflow_dispatch:
+  pull_request_target:
```
Comment (alemorcuq) [inline .github/workflows/helm-vib-lint.yaml:4]:
```suggestion
  pull_request_target:
    branches:
      - main
      - bitnami-labs:main
```

This is how it was before and I think it is fine. Only run on PRs whose target is `main` or `bitnami-labs:main`

Comment (josvazg) [reply .github/workflows/helm-vib-lint.yaml:4]: ok

File: .github/workflows/helm-vib-lint.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    branches:
+      - main
+      - bitnami-labs:main
+    paths:
+       - 'helm/**'
```
Comment (alemorcuq) [inline .github/workflows/helm-vib-lint.yaml:9]:
```suggestion
  pull_request_target:
    branches:
       - main
       - bitnami-labs:main
    paths:
       - 'helm/**'
```

Alignment

Comment (josvazg) [reply .github/workflows/helm-vib-lint.yaml:9]: You are right, but the misalignment is on the paths I think.

```suggestion
  pull_request_target:
    branches:
      - main
      - bitnami-labs:main
    paths:
      - 'helm/**'
```

**Review by alemorcuq — APPROVED** [review]

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**